### PR TITLE
Restored Mobile Functionality

### PR DESCRIPTION
**Pull request by TensionCoding**
Touch events no longer worked. I looked into the [ParentNode.md](https://github.com/open-source-labs/Svelvet/blob/main/NPM%20Package/svelvet/Future%20Iteration/ParentNode.md) and found that the original Svelvet 2.0 code was uncommented and the Svelvet 4.0 code was commented out. However, in the actual Npm package I found the opposite. I added the 2.0 code and commented out the 4.0 code for future contributors to work on.
Resolving issue #140.

### CODE REVIEW

**Comment by aubertlone** [comment]
Hey Walter,

Thank you for your continued work on Svelvet! Bringing in the changes from your commit would lose the node grouping functionality. But of course, the node grouping functionality doesn't really mean anything if onTouchMove doesn't work.

I've looked at your commit, it makes total sense to me about restoring the previous logic from onTouchMove from an earlier version. I'd like to review & approve your commit, but unfortunately the Cypress tests are failing with this PR. For us, it was often due to mismatches on package-lock.json. Running npm install --> pushing commit --> PR solved those issues for us when we had some cypress tests fail the morning of launch. The cypress tests passed the new PR after those steps.

Also, let me get in touch with the team member who specifically wrote that logic to see if there's any way to retain node grouping functionality on mobile. Of course, restoring the touch features is the priority.

In summary, I'd be happy to approve/review & merge into main if the cypress tests pass. Even if we lose the node grouping functionality on mobile, the priority is restoring onTouchMove to a working state.

Thanks again for keeping an eye on this project!!

**Comment by TensionCoding** [comment]
I believe the test failed because I am using an outside repo to make a PR that doesn't have access to the secrets stored in GitHub actions. The error you received in your earlier was related to a Npm issue, this Cypress test failed because of:

Error: supabaseUrl is required.

I will make a branch off of this repo and it will likely work. Trying now

**Comment by TensionCoding** [comment]
Closing PR.
PR #143 passed all checks when performed inside this repo
